test/asami/api_test.cljc

```diff
@@ -15,6 +15,16 @@
   #?(:clj (:import [clojure.lang ExceptionInfo]
                    [java.time Instant])))
 
+(def ^:dynamic *conn* nil)
+
+(use-fixtures :each (fn init-test-conn+cleanup-all [test]
+                      (binding [*conn* (a/connect "asami:mem://apitest")]
+                        (try (test)
+                             (finally
+                               (a/delete-database "asami:mem://apitest")
+                               (doseq [url (keys @a/connections)]
+                                 (a/delete-database url)))))))
+
```

Comment:
This works, but was there a reason why?

The existing approach does require a new name for everything (which is a pain if you copy/paste and forget to do this), but it also let me inspect everything after the tests were done. This was especially important in the API tests on durable storage

Reply:
FYI I have reverted the change not to block the PR. But it would still be nice to have a way to run tests repeatedly in the REPL.